test/io-tests.sh

```diff
@@ -36,17 +36,71 @@ trap cleanup sigint sigterm exit
 # Port for Test PostgREST Server (must match config)
 pgrPort=49421 # in range 49152–65535: for private or temporary use
 
+# Colors
+NC='\033[0m' # no color
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+
 # TAP utilities
 currentTest=1
 failedTests=0
 bailOut(){ echo "Bail out! $1"; exit 1; }
-result(){ echo "$1 $currentTest $2"; currentTest=$(( $currentTest + 1 )); }
-todo(){ result 'ok' "# TODO: $*"; }
-skip(){ result 'ok' "# SKIP: $*"; }
-ok(){ result 'ok' "- $1"; }
-ko(){ result 'not ok' "- $1"; failedTests=$(( $failedTests + 1 )); }
+result(){ echo -e "$1 $currentTest $2${NC}"; currentTest=$(( $currentTest + 1 )); }
+todo(){ result "${YELLOW}ok" "# TODO: $*"; }
+skip(){ result "${YELLOW}ok" "# SKIP: $*"; }
+ok(){ result "${GREEN}ok" "- $1"; }
+ko(){ result "${RED}not ok" "- $1"; failedTests=$(( $failedTests + 1 )); }
 comment(){ echo "# $1"; }
 
+####################
+# FOREGROUND TESTS #
```

Comment:
appreciate suggestions for a better name for this section and the "background" section. The distinction here is, that for those tests `postgrest` immediately returns and is not sent to the background.

Maybe it's just not needed at all. But it does give a bit of structure to that file. I'm thinking about running shellcheck against it.... but I am bit afraid of the result as well :D

Reply:
Hmm.. I also don't have a better name, maybe just put the distinction as a comment?